database-backups/.gitignore

```diff
@@ -0,0 +1,5 @@
+# ignore everything except .gitignore and encrypted files (*.gpg) and scripts
+*
+!.gitignore
+!*.gpg
+!*.sh
```

Comment:
Are you expecting to add scripts (`*.sh`) to this directory? I could see possibly checking in encrypted files (`*.gpg`) although for backups they're probably too big to want them in the repo.

Reply:
I just added this from the snippet you sent over slack. I guess that was just an example? I thought the idea was to possibly add the .sql backups here, but only if they were encrypted. So I guess for now, ignore everything in this folder except for the gitignore?

Reply:
I'm pretty sure I said "here is an example".
So I was asking for confirmation that you thought about what was in the gitignore when you added it (or not), and the reasoning behind what you did.
If I had said, here use this, I would expect you to look at it and understand what it does, or otherwise to ask the same questions I just asked you.
Potentially allowing encrypted backups to be in this folder is very reasonable, and a good reason to keep the `!*.gpg`, although if you do that, you should probably not try to restore those files in your restore script.

Reply:
Ok makes sense.